test/unit/server.spec.js

```diff
@@ -120,10 +118,10 @@ describe('server', () => {
       close: sinon.spy((callback) => callback && callback())
     }
 
-    sinon
-      .stub(server._injector, 'get')
-      .withArgs('webServer').returns(mockWebServer)
-      .withArgs('socketServer').returns(mockSocketServer)
+    const injectorStub = sinon.stub(server._injector, 'get')
+    injectorStub.withArgs('socketServer').returns(mockSocketServer)
+    injectorStub.withArgs('webServer').returns(mockWebServer)
+    injectorStub.callThrough()
```

Comment:
Sinon would override stub behavior with the last call if chained, so have to add each behavior as a separate statement.